app/Dockerfile

```diff
@@ -4,34 +4,123 @@
 # The build stage that will be used to deploy to the various environments
```

Comment:
the changes here to productionize the docker image were copied from copied from https://github.com/navapbc/template-application-flask/pull/141/files